### DISCUSSION TITLE
I'm Confused On How To Install The Linux Version

### DISCUSSION THREAD

**Discussion by Reed-Mel** [vgmtrans/vgmtrans#228] [Q&A] [answered]
How do you get the Linux app file? The page has no hints and Google isn't helping.

**Comment by piratesephiroth** [comment, marked as answer]
click the link in the readme (or the "Actions" tab above), then click on a workflow run and you'll see the artifacts at the bottom

**Reply by Reed-Mel** [reply]
Thanks a lot!
